### PR TITLE
Cuda availability check and error messages

### PR DESCRIPTION
**Pull request by hhsecond**
Quick fix for #152 & #155 (Pending the GPU check for ONNXRuntime)

### CODE REVIEW

File: src/backends/onnxruntime.c [hunk 1]
```diff
@@ -270,6 +270,12 @@ RAI_Model *RAI_ModelCreateORT(RAI_Backend backend, RAI_Device device, int64_t de
   if (device == RAI_DEVICE_GPU) {
     OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, deviceid);
   }
+#else
```
Comment (hhsecond) [inline src/backends/onnxruntime.c]: `RAI_ONNXRUNTIME_USE_CUDA` doesn't seem to propagate with `-DDEVICE=gpu` in cmake

Comment (hhsecond) [reply src/backends/onnxruntime.c]: Rearranged the CMAKE file. It doesn't seem to break anything else. Please let me know

**Comment by hhsecond** [comment]
Rebased and force pushed

**Comment by hhsecond** [comment]
Tests are failing by the rltest bug which is being tracked here https://github.com/RedisLabsModules/RLTest/issues/52

File: src/backends/onnxruntime.c (hunk 1 above)
Comment (hhsecond) [reply src/backends/onnxruntime.c]: Changed the order in CMAKE file. It should not break anything else but appreciate if you could take a look

**Comment by lantiga** [comment]
Hi Sherin, can you confirm tests are passing locally on a GPU machine?

**Comment by hhsecond** [comment]
@lantiga Tests should pass ideally. But after the recent changes in RLTest, all our existing tests are failing -> `con.execute_command` is not returning bytes strings anymore. I'll fix our tests soon and then we are good to go

**Comment by hhsecond** [comment]
Changes in python Redis client has been reverted back and Circle tests are not failing because of those changes. But there is still one issue with load backend. Let me figure that out and then we can merge

**Comment by hhsecond** [comment]
@lantiga Everything looks good in the CI

**Comment by lantiga** [comment]
Great, thanks!

**Pull request merged**